### DISCUSSION TITLE
[Feature] Add possibility to import pictures from a partner to our library

### DISCUSSION THREAD

**Discussion by githubbab** [immich-app/immich#5359] [Feature Request]
### The feature

Add possibility to import pictures from a partner to our library.
Or add shared pictures when we put them in Album.
After, if partners remove pictures we ca keep them.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by AngelaDMerkel** [comment]
I agree. This would be consistent with the feature in Google Photos and in my experience is the best way to handle this problem. Once shared it should be possible to selectively add shared photos to your timeline without needing to merge every image as in partner sharing.

**Comment by aisbergde** [comment]
not only from partners, but from shared albums in General.

**Reply by ashleycollinge1** [reply]
Yes, I want family to be able to add pictures from my shared album to their own album in their account

**Comment by maarten-vdb** [comment]
This would make using immich as a familty way better! Right in my own setting immich only works for my own purposes

**Comment by petersuba** [comment]
Isn't this overlapping or the same as #7047 ?

**Comment by bo0tzz** [comment]
#1587
